### ISSUE TITLE
[sct_register_multimodal] Columnwise should force type=seg

### ISSUE THREAD

**Issue by tanguyduval** [neuropoly/spinalcordtoolbox#1034]
Currently the option
``````
step=1,algo=columnwise
``````

Assumes type=im:

``````
step=1,type=im,algo=columnwise
`````

**Comment by jcohenadad** [comment]
it's not a bug-- user should specify what to use with each algo.

i agree it would be better to warn the user of course.

**Comment by tanguyduval** [comment]
I see, maybe we should look at the warper:
- make step and type mandatory options
- check the inputs. For instance if you do type=fdsiojfo, you need to wait the function to run until you see a bug: ````ERROR: Wrong image type````

**Comment by SaraDupont** [comment]
The param 'type' is now mandatory (see PR #1048) 
Regarding the 'step' param, it's delicate since the function needs a default param that can be overwritten by the user's param, so `step=''` doesn't work.

**Comment by SaraDupont** [comment]
Fixed in PR #1057
